recipes/webcolors/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.5" %}
+
+package:
+  name: webcolors
+  version: {{ version }}
+
+source:
+  fn: webcolors-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
+  md5: 2e9da1bbb7a087c6108e05be64640afb
+
+build:
+  number: 0
+  script:
+    - source activate _build  # [unix]
+    - activate _build         # [win]
```

Comment:
Don't forget you can log into appveyor directly:
https://www.appveyor.com/docs/how-to/rdp-to-build-worker

On Mon, Jun 6, 2016 at 4:15 PM jakirkham <notifications@github.com> wrote:

> In recipes/webcolors/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/642#discussion_r65970568>
> :
>
> > +{% set version = "1.5" %}
> > +
> > +package:
> > +  name: webcolors
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: webcolors-{{ version }}.tar.gz
> > +  url: https://pypi.python.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
> > +  md5: 2e9da1bbb7a087c6108e05be64640afb
> > +
> > +build:
> > +  number: 0
> > +  script:
> > +    - source activate _build  # [unix]
> > +    - activate _build         # [win]
>
> Though I do have a sort of working Windows machine (think the compiler
> installs didn't go over well 😨). Fortunately, this doesn't require
> working compilers (I would hope 👼). FWICT it is not activating correctly
> there. 😕
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/642/files/013ca3f8d866a57c79724e8ecb99a0c4b38c326d#r65970568>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACV-bkgYdk7ebR2-bae5D8HTPEdBksLks5qJI3qgaJpZM4Ih7wZ>
> .
>